.github/workflows/ci-nightly.yml

```diff
@@ -12,6 +12,8 @@ jobs:
     runs-on: ubuntu-latest
 
     env:
+      NPM_PAT: ${{ secrets.PACK_TOKEN }}
```

Comment:
PACK_TOKEN should be reverted back to the correct me for our secret npm token name.